src/Rating.jsx

```diff
@@ -158,6 +160,7 @@ class Rating extends React.PureComponent {
         style={{ display: 'inline-block', direction }}
         onMouseEnter={!readonly ? this.onMouseEnter : noop}
         onMouseLeave={!readonly ? this.onMouseLeave : noop}
+        {...other}
```

Comment:
Please remove passing other props. Let's open an issue to discuss any specific additions you'd like to see to the API. I don't really like the idea of mass passing of undocumented props.